.github/workflows/contributor-doc.yml

```diff
@@ -21,7 +21,7 @@ permissions: {}
 jobs:
   deploy:
     permissions:
-      contents: write  #  to push pages branch (peaceiris/actions-gh-pages)
+      contents: write #  to push pages branch (peaceiris/actions-gh-pages)
```

Comment:
Not sure why I have this diff for github workflows 🤷. I'm assuming husky made these changes when I committed after merging and resolving conflicts but not sure how they made it on main...